src/graphql/query/thanksPage.graphql

```diff
@@ -146,6 +147,10 @@ query checkoutReceipt($checkoutId: Int!, $visitorId: String) {
 			key
 			value
 		}
+		newThanksPageEnabled: featureSetting(key: "new_thanks_page.enabled") {
```

Comment:
Is there a reason we're using a feature setting instead of an experiment setting?